functions/go/search-replace/README.md

```diff
@@ -37,9 +38,18 @@ value of the field by default without requiring start (^) and end ($) characters
 
 by-path
 Match by path expression of a field. Path expressions are used to deeply navigate
-and match particular yaml nodes. Please note that the path expressions are not
+and match particular yaml nodes. Please note that the field path expressions are not
 regular expressions.
 
+by-file-path
```

Comment:
Sorry, don't intend to derail the PR, but this overlaps with the filtering functionality we plan to introduce in the orchestrator, right ? I think filtering `by-file-path` will be useful as a general filter. We should probably wait ?

Reply:
We discussed this offline and decided it doesn't harm to include this filter at the function level. so going forward with the PR.